ReIndexOrchestrator/index.js

```diff
@@ -0,0 +1,44 @@
+const df = require('durable-functions');
+const moment = require('moment');
+
+module.exports = df.orchestrator(function* orchestratorFunctionGenerator(context) {
+  context.log('Starting ReIndexOrchestrator');
+  const input = context.df.getInput();
+
+  context.log({ input });
```

Comment:
Could some additional information be added to increase the usefulness of this message within the logs please?